src/protocols/secure_channel/PASESession.h

```diff
@@ -76,10 +76,24 @@ struct PASESessionSerializable
     uint16_t mPeerSessionId;
 };
 
+typedef uint8_t PASEVerifierSerialized[2 * kSpake2p_WS_Length];
+
 struct PASEVerifier
 {
     uint8_t mW0[kSpake2p_WS_Length];
     uint8_t mL[kSpake2p_WS_Length];
+
+    /** @brief Serialize the PASE Verifier.
+     *
+     * @return Returns a CHIP_ERROR on error, CHIP_NO_ERROR otherwise
```

Comment:
delete these @return comments - it is obvious that returning a chip error means 'return error on success and no-error on failure')

Reply:
removed